pytest_ethereum/linker.py

```diff
@@ -42,12 +42,20 @@ def _deploy(contract_name: str, args: Any, package: Package) -> Tuple[Package, A
 
     # Deploy new instance
     factory = package.get_contract_factory(contract_name)
+    if not factory.runtime_link_refs and factory.deployment_link_refs:
+        raise LinkerError(
+            "Contract factory: {0} is missing runtime link references, which are necessary "
```

Comment:
If this is python3.6 only you can change these to be f-strings which recently became our defacto formatting method in py-evm/trinity

https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python